core-plugins/src/test/java/co/cask/hydrator/plugin/batch/ETLSnapshotTestRun.java

```diff
@@ -141,27 +149,31 @@ public void testMultiSnapshotOutput() throws Exception {
   }
 
   // deploys a pipeline that reads using a snapshot source and checks that it writes the expected records.
-  private void testSource(String sourcePlugin, String sourceName, Map<String, Integer> expected) throws Exception {
+  private void testSource(String sourceETLPlugin, String sourceName, Map<String, Integer> expected) throws Exception {
     // run another pipeline that reads from avro dataset
     ETLStage source = new ETLStage(
       "source",
-      new Plugin(sourcePlugin, ImmutableMap.<String, String>builder()
-        .put(Properties.Table.NAME, sourceName)
-        .put(Properties.Table.PROPERTY_SCHEMA, SCHEMA.toString())
-        .put(Properties.Table.PROPERTY_SCHEMA_ROW_FIELD, "id")
-        .build()));
+      new ETLPlugin(sourceETLPlugin, BatchSource.PLUGIN_TYPE,
+                    ImmutableMap.<String, String>builder()
```

Comment:
not a big deal - seems like we use the builder pattern everywhere else in these test cases.